packages/mds-jurisdiction/types.ts

```diff
@@ -0,0 +1,74 @@
+/*
+    Copyright 2019-2020 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import { ApiRequest, ApiResponse, ApiResponseLocals } from '@mds-core/mds-api-server'
+import { UUID, Jurisdiction } from '@mds-core/mds-types'
+import { Params, ParamsDictionary } from 'express-serve-static-core'
+import { CreateJurisdictionType } from '@mds-core/mds-jurisdiction-service'
+
+// Place newer versions at the beginning of the list
+const JURISDICTION_API_VERSIONS = ['0.1.0'] as const
```

Comment:
Can we have the most recent version pull from the package.json?

Reply:
The package version is different than the API/protocol version.